estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,59 @@
+from odoo import api, exceptions, fields, models
+from dateutil.relativedelta import relativedelta
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "My Estate Property Offer"
+    _order = "price desc"
+
+    price = fields.Float(string = "Price")
+    status = fields.Selection([('accept', 'Accept'), ('refused', 'Refused')], copy = False, string = "Status")
+    partner_id = fields.Many2one('res.partner', string = "Partner", required = True)
+    property_id = fields.Many2one('estate.property', string = 'Property', required = True)
+    property_type_id = fields.Many2one(related = 'property_id.property_type_id')
+
+    validity = fields.Integer(string = "Validity", default = 7)
+    date_deadline = fields.Date(compute = "_compute_deadline", inverse = "_inverse_deadline", string = "Deadline")
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            property = self.env['estate.property'].browse(vals['property_id'])
```

Comment:
will crash if property_id is not set
even if it's a required field, it should not crash because of your code. The ORM will do the job.